internal/rbd/nodeserver.go

```diff
@@ -149,6 +149,65 @@ func healerStageTransaction(ctx context.Context, cr *util.Credentials, volOps *r
 	return nil
 }
 
+// populateVolOptions update the fields in rbdVolume struct based on the request it received.
+func populateRbdVol(
+	ctx context.Context,
+	req *csi.NodeStageVolumeRequest,
+	disableInUseChecks bool,
+	volID string, cr *util.Credentials) (*rbdVolume, error) {
+	var err error
+	var j *journal.Connection
+	volOptions, err := genVolFromVolumeOptions(ctx, req.GetVolumeContext(), req.GetSecrets(), disableInUseChecks)
```

Comment:
`disableInUseChecks` is used only in genVolFromOptions.  we can move the calculation of `disableInUseChecks` to this function it no need to be an argument.

Reply:
Done..